### DISCUSSION TITLE
Issue while using git config --global user.name and git config --global user.email

### DISCUSSION THREAD

**Discussion by itsaalexk** [community/community#38505] [New to GitHub]
I have just noticed when setting up my Github account that  when you type git config --global user.name  "your name" and git config --global user.email you@example.com. The email example SHOULD include quote marks, this is likely a typo that can be fixed pretty easy , instead it should be like this => git config --global user.email "YOU@EXAMPLE.COM".

I was having some issues myself to set up my account because I didn't write my email inside the quote marks and was having a hard time setting it up , so to avoid future conflicts I suggest that you fix that so its easier to understand

![example](https://user-images.githubusercontent.com/106255458/200409790-36decad7-d66d-4e19-85e8-e86e6bc4d1a1.png)

As you can see on the image , the user.name appears to be quoted while user.email is not ,  so this can lead into a confusion.
Instead it should be like that.

`git config --global user.name "your name"   
git config --global user.email "you@example.com"`

**Comment by ossanna16** [comment]
Hi there @itsaalexk and welcome to our community! Thank you for the feedback!

**Reply by itsaalexk** [reply]
Thank You!
